traffic_ops/app/lib/API/User.pm

```diff
@@ -159,6 +161,9 @@ sub update {
 		return $self->not_found();
 	}
 
+	#setting tenant_id to undef if tenant is not set. TODO(nirs): remove when tenancy is no longer optional in the API
+ 	my $tenant_id = exists($params->{tenantId}) ? $params->{tenantId} :  undef; 
```

Comment:
Indeed it will be sufficient. Still I prefer to be explicit so the code documents itself.
Can I leave it that way?